envergo/evaluations/views.py

```diff
@@ -311,6 +311,14 @@ def reset_data(self):
         self.request.session.pop(DATA_KEY, None)
         self.request.session.modified = True
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        form = self.get_form(self.get_form_class())
+        if form.is_bound and not form.is_valid():
+            # has it is not a complete url, Matomo concatenate it to the current url, which is OK for us
```

Comment:
:eyes: 

```suggestion
            # as it is not a complete url, Matomo concatenate it to the current url, which is OK for us
```

Je ne savais pas ça. Cool.